plugins/less.js

```diff
@@ -26,25 +26,24 @@ module.exports = function (options) {
     options.ext = options.ext || 'less';
 
     return function (data, args, callback) {
-        var parser = new(lib.Parser)({
+        var options = {
             paths: args.paths,
             filename: args.context.name,
             dumpLineNumbers: 'comments'
-        });
+        };
 
         try {
             // Really? REALLY?! It takes an error-handling callback but still can throw errors?
-            parser.parse(data.toString('utf8'), function (err, tree) {
-                if (err) {
-                    callback(err);
-                    return;
-                }
-                callback(null, tree.toCSS());
-            });
+            lib.render(data.toString('utf8'), options)
+              .then(function(tree) {
+                callback(null, tree.css);
```

Comment:
Tree is an object, `tree.css` Is a string, so I don't know why you would return two different types of values for consistencies sake...